src/main/resources/acls/agent-group.ttl

```diff
@@ -1,5 +1,5 @@
 @prefix    acl:  <http://www.w3.org/ns/auth/acl#>.
 @prefix  vcard:  <http://www.w3.org/2006/vcard/ns#>.
 <>  a                vcard:Group;
-    vcard:hasMember  "${user}";
+    vcard:hasMember  "${user}" .
```

Comment:
This file does not seem related to the failing test.

Reply:
oh - you're right.  I'm not sure how that got in there.  I'l remove.